### PR TITLE
vktrace: 32 bit build fix

### PR DESCRIPTION
**Pull request by johnzupin**
Problem was discovered in GH issue #371

vktrace/CMakeList.txt: fix conditional on 32bit build check 

vktrace/vktrace_layer/CMakeLists.txt: change cmake function call to
find_library

### CODE REVIEW

**Comment by johnzupin** [comment]
Karl I was told to assign this to you. If @davidlunarg or @joey-lunarg approve this I can probably merge it.

**Comment by johnzupin** [comment]
This might be poorly named PR, the second part i changed
```
-       pkg_search_module(XCB-KEYSYMS REQUIRED xcb-keysyms)
+       find_library(XCB-KEYSYMS REQUIRED xcb-keysyms)
```
in vktrace/vktrace_layer/CMakeLists.txt.

It initially couldn't find the XCB-KEYSYMS library until I changed it to find_library.

File: vktrace/vktrace_layer/CMakeLists.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 2.8)
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
    if (BUILD_WSI_XCB_SUPPORT)
        set(ENV{VULKAN_WSI} "Xcb")
-       pkg_search_module(XCB-KEYSYMS REQUIRED xcb-keysyms)
+       find_library(XCB-KEYSYMS REQUIRED xcb-keysyms)
```
Comment (karl-lunarg) [inline vktrace/vktrace_layer/CMakeLists.txt]: find_package  is probably preferred here.   find_package will set vars that help CMake locate headers and libraries, whereas find_library will set vars that help CMake locate just the libraries.

I'd suggest trying find_package and use it if it works.

**Comment by johnzupin** [comment]
@karl-lunarg  I did something a bit different, I put the job of finding libxcb-keysyms in FindXCB.cmake. FindXCB.cmake looks for components and since libxcb-keysyms1-dev is a required package in the docs I was thinking that's were it should be. 

I did have to change a line in FindXCB.cmake to look for "xcb_keysyms.h" instead of "keysyms.h". That seems to be what the header file is called at least on Ubuntu.

Let me know if this is wrong.

**Comment by karl-lunarg** [comment]
I'd be afraid of the cmake/findXCB.cmake change not working on other distros.

I just tried it with the change in the conditional and changing pkg_search_module to find_library, and it worked.  Plus the OP said that find_library worked.  So I'd favor the find_library change.

**Comment by johnzupin** [comment]
@karl-lunarg I made these changes and the tests pass. Let me know if you approve.

**Pull request merged**